src/sentry/static/sentry/app/components/events/eventDataSection.tsx

```diff
@@ -3,7 +3,26 @@ import React from 'react';
 import {t} from 'app/locale';
 import GuideAnchor from 'app/components/assistant/guideAnchor';
 
-class EventDataSection extends React.Component {
+const defaultProps = {
+  wrapTitle: true,
+  raw: false,
+  hideGuide: false,
+  toggleRaw: (_enable: boolean) => void 0,
+};
+
+type DefaultProps = Readonly<typeof defaultProps>;
+
+type Props = {
+  className?: string;
+  title: React.ReactText;
+  type: string;
+  wrapTitle: boolean;
+  toggleRaw: Function;
```

Comment:
@markstory any reason you don't use a function signature here?

Reply:
I'm dumb. This doesn't need to be here it is also defined in DefaultProps.